language-support/ts/daml-ledger/index.ts

```diff
@@ -1378,6 +1389,11 @@ class Ledger {
     return decode(jtv.array(partyInfoDecoder), json);
   }
 
+  async getUser(): Promise<User> {
```

Comment:
I think the other functions and docs have doc comments, maybe we should also add some here?